Sources/CacheContainer.swift

```diff
@@ -46,3 +46,68 @@ public final class CacheContainer {
         }
     }
 }
+
+final class ThreadSafeDictionary<V: Hashable,T>: Collection {
+    private var dictionary: [V: T]
+    private let concurrentQueue = DispatchQueue(label: "Dictionary Barrier Queue",
+                                                attributes: .concurrent)
+
+    var keys: Dictionary<V, T>.Keys {
+        self.concurrentQueue.sync {
+            return self.dictionary.keys
+        }
+    }
+
+    var values: Dictionary<V, T>.Values {
+        self.concurrentQueue.sync {
+            return self.dictionary.values
+        }
+    }
+
+    var startIndex: Dictionary<V, T>.Index {
+        self.concurrentQueue.sync {
+            return self.dictionary.startIndex
+        }
+    }
+
+    var endIndex: Dictionary<V, T>.Index {
+        self.concurrentQueue.sync {
+            return self.dictionary.endIndex
+        }
+    }
+
+    init(dict: [V: T] = [V:T]()) {
```

Comment:
Please unify the format of dictionary type declaration.
```suggestion
    init(dict: [V : T] = [V : T]()) {
```

Reply:
Fixed by [7c1ad17](https://github.com/cybozu/CachePropertyKit/pull/2/commits/7c1ad1725563f67849a27838b787a38b83b4590c).